src/mappings/rewardsStaking.ts

```diff
@@ -0,0 +1,46 @@
+import { TypedEvent } from '@subql/contract-sdk/typechain/common';
+import { EthereumLog, EthereumResult } from '@subql/types-ethereum';
+import { BigNumber } from 'ethers';
+import { IndexerStakeWeight } from '../types';
+import { biToDate } from './utils';
+import { getCurrentEra } from './eraManager';
+
+// TODO: this is a temporary solution to avoid type errors
+export interface RunnerWeightAppliedEventObject {
```

Comment:
Is it because the abi you used doesn't contain latest change?

Reply:
yes, with contract sdk v1.1.0